packages/algorand-js/src/types.ts

```diff
@@ -11,6 +11,109 @@ export type AppArgs = Array<string | number>;
+  AssetSender, // 32 byte address. Causes clawback of all value of asset from AssetSender if Sender is the Clawback address of the asset.
+  AssetReceiver, // 32 byte address
+  AssetCloseTo, // 32 byte address
+  GroupIndex, // Position of this transaction within an atomic transaction group. A stand-alone transaction is implicitly element 0 in a group of 1
+  TxID, // The computed ID for this transaction. 32 bytes.
+  ApplicationID, // ApplicationID from ApplicationCall transaction.
+  OnCompletion, // ApplicationCall transaction on completion action.
+  ApplicationArgs, // Arguments passed to the application in the ApplicationCall transaction.
+  NumAppArgs, // Number of ApplicationArgs.
+  Accounts, // Accounts listed in the ApplicationCall transaction.
+  NumAccounts, // Number of Accounts
+  ApprovalProgram, // Approval program
+  ClearStateProgram, // Clear state program
+  RekeyTo, // 32 byte Sender's new AuthAddr
+  ConfigAsset, // Asset ID in asset config transaction
+  ConfigAssetTotal, // Total number of units of this asset created
+  ConfigAssetDecimals, // Number of digits to display after the decimal place when displaying the asset
+  ConfigAssetDefaultFrozen, // Whether the asset's slots are frozen by default or not, 0 or 1
+  ConfigAssetUnitName, // Unit name of the asset
+  ConfigAssetName, // The asset name
+  ConfigAssetURL, // URL
+  ConfigAssetMetadataHash, // 32 byte commitment to some unspecified asset metadata
+  ConfigAssetManager, // 32 byte address
+  ConfigAssetReserve, // 32 byte address
+  ConfigAssetFreeze, // 32 byte address
+  ConfigAssetClawback, // 32 byte address
+  FreezeAsset, // Asset ID being frozen or un-frozen
+  FreezeAssetAccount, // 32 byte address of the account whose asset slot is being frozen or un-frozen.
+  FreezeAssetFrozen // The new frozen value, 0 or 1
+}
+
+export enum TxnType {
+  unknown, // Unknown type. Invalid transaction
+  pay, // Payment
+  keyreg, // KeyRegistration
+  acfg, // AssetConfig
+  axfer, // AssetTransfer
+  afrz, // AssetFreeze
+  appl // ApplicationCall
+}
+
+// https://developer.algorand.org/docs/reference/teal/specification/#oncomplete
+export enum TxnOnComplete {
+  NoOp,
+  OptIn,
+  CloseOut,
+  ClearState,
+  UpdateApplication,
+  DeleteApplication
+}
+
+export enum GlobalField {
+  MinTxnFee, // micro Algos
+  MinBalance, // micro Algos
+  MaxTxnLife, // rounds
+  ZeroAddress, // 32 byte address of all zero bytes
+  GroupSize, // Number of transactions in this atomic transaction group. At least 1
+  LogicSigVersion, // Maximum supported TEAL version.
+  Round, // Current round number
+  LatestTimestamp, // Last confirmed block UNIX timestamp. Fails if negative.
+  CurrentApplicationID // ID of current application executing. Fails if no such application is executing.
+}
+
+export enum AssetHolding {
+  AssetBalance, // Amount of the asset unit held by this account
+  AssetFrozen // Is the asset frozen or not
+}
+
+export enum AssetParam {
```

Comment:
let's add a comment here `this is for global storage`
and above type will be mapped to specific account.

Reply:
Yes, we should have docstrings wherever possible

Reply:
added.